### DISCUSSION TITLE
Meatloaf looks cool!

### DISCUSSION THREAD

**Discussion by mwedmark** [idolpx/meatloaf-archived#7] [General]
This project looks cool. Does it use ESP8266 or ESP32? From looking into the code it looks like both have been tried. 
From the looks of pictures it looks like the Wemos D1 Mini development board is used and that hosts a ESP8266? 

Either way I would like to try this out. Is there a PCB created and shared /(gerber or KiCad fles) for Meatloaf?
I guess there is no chance of Meatloaf being able to do full emulation for fast loader support even on a ESP32 or?

**Comment by idolpx** [comment]
My goal was to push the limits of the ESP8266 but also port it to work on ESP32.  A lot of progress has been made in that area already.  You can use any dev board but I used a D1 MINI Pro with 16MB of SPI Flash.  That is enough space for roughly 120 D64 floppy images.  Main goal is to have stuff accessible via the internet but it is nice to have a few essential programs cached for offline use.

I plan to have fastloader support for Meatloaf at the SD2IEC level minimum but have ideas for making it even better.

The ESP32 is powerful enough to do full emulation.  See my repo for a port of Pi1541 that I've already started.  It has a ways to go but I believe it is totally doable.

**Reply by mwedmark** [reply]
Sounds good! Is there are working file browser that work for http access and makes it easy to browse? I see fb64 but does that work only for flash storage or http? If not maybe put up some simple tasks/issues/features and I can help out when I've built mine. 25 years of professional development in c/c++, c#, web, assembler, vb

**Reply by idolpx** [reply]
So sorry for not seeing this sooner.  Would love to get some help with stuff like that.  Please join us in Discord to discuss.

**Reply by mwedmark** [reply]
Thanks for feedback on this old issue.  My name is overclocked on discord I think you know me!

**Comment by dcs-electronics** [comment]
I have just purchased a C64 (I do love old 8 bit stuff) for the express purpose of getting it online.How is progress going - I will build some boards from your details soon.I am retired so now do electronics to keep the brain going and have a little website selling the odd bits and pieces , blank pcb's for projects etc.Any help on getting going with fujinet etc would be fantastic.
